package.json

```diff
@@ -240,7 +241,7 @@
   },
   "husky": {
     "hooks": {
-      "post-merge": "node tools update"
+      "post-merge": "npm install"
```

Comment:
Why did you need to switch?

Reply:
this is updating packages every time i switch branches, which is becoming more of an annoyance than something helpful. we're going to start auto-merging dependabot PRs once we have end to end tests (next sprint) so I'm not too concerned about it